### ISSUE TITLE
Error 'ERROR: You're using an RSA key with SHA-1' when trying to push to remote.

### ISSUE THREAD

**Issue by safield** [Murmele/Gittyup#293]
```
Unable to push to 'origin' - ERROR: You're using an RSA key with SHA-1, which is no longer allowed. Please use a newer client or a different key type.
Please see https://github.blog/2021-09-01-improving-git-protocol-security-github/ for more information.
```
Recieved this error when trying to push to remote in Gittyup. Opened the same repo in other git clients and the repo pushes fine.

**Comment by exactly-one-kas** [comment]
This is a known issue which will be fixed with the 1.2.0 release (coming soon™)
In the meantime, you can try the [current pre-release](https://github.com/Murmele/Gittyup/releases/tag/latest)

**Comment by exactly-one-kas** [comment]
With 1.2.0 being released, this is now fixed in the stable version
